src/assets/scss/modules/_forms.scss

```
@@ -450,7 +456,8 @@
 
     .password-message {
       margin: var.$dp-spaces--lv0;
-      font-size: var.$dp-sizing--lvl2;
+      /* Agregar variables de fonts-size cuando se modifiquen a rem */
```

Si utilizás la palabra clave `TODO: `, algunos IDEs te encuentran más fácilmente los items pendientes.

```suggestion
      /* TODO: Agregar variables de fonts-size cuando se modifiquen a rem */
```

---

👍 lo agrego!